### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by IceCodeNew**

### CODE REVIEW

**Comment by IceCodeNew** [comment]
Sorry for the force push on the master branch, I had pushed to the master branch directly. Considering all of the options, I decided to falsify the history.

BTW please consider protecting the master branch ;-)

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -58,7 +60,7 @@ jobs:
 
     - name: Build
       run: |
-        xcaddy build master --output ${{ runner.temp }}/${{ matrix.name }} --with github.com/mholt/caddy-l4=./
+        xcaddy build latest --output ${{ runner.temp }}/${{ matrix.name }} --with github.com/mholt/caddy-l4=./
```
Comment (francislavoie) [inline .github/workflows/go.yml:63]: I don't think this is right, `latest` isn't a git ref that exists.

Comment (mohammed90) [reply .github/workflows/go.yml:63]: `latest` is a special case of the `go {get,mod}` resolver which points to the latest tagged release. This is specified here:
https://go.dev/ref/mod#version-queries

Comment (IceCodeNew) [reply .github/workflows/go.yml:63]: Thanks for pointing that out, I was narrowing down what is causing the CI failed. So the change here won't get into the final PR.

BTW I believe the `latest` would be recognized as a keyword, refering to https://github.com/caddyserver/xcaddy/blob/f42cc51e4bad735f1e23616109012b56217c72f0/README.md?plain=1#L68

Comment (IceCodeNew) [reply .github/workflows/go.yml]: @francislavoie @mohammed90 Thanks for taking the time to review this PR. This change of the `xcaddy build latest` instruction has been reverted.

Comment (francislavoie) [reply .github/workflows/go.yml]: Oh right, my bad, forgot we had that :+1:

**Review by francislavoie — APPROVED** [review]

**Pull request merged**